templates/partials/_code-snippet.html

```diff
@@ -1,4 +1,3 @@
-<div class="p-code-copyable">
-  <input class="p-code-copyable__input" id="snippet-{{ snippet_id }}" value="{{ snippet_value }}" readonly="readonly">
-  <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-{{ snippet_id }}">Copy to clipboard</button>
+<div class="p-code-snippet">
+  <pre class="p-code-snippet__block--icon" style="white-space: pre-wrap;"><code>{{ snippet_value }}</code></pre>
```

Comment:
Using inline style to set wrapping may be harder to find and fix later.
```suggestion
  <pre class="p-code-snippet__block--icon is-wrapped"><code>{{ snippet_value }}</code></pre>
```
I would suggest adding `is-wrapped` class (which is gonna be supported in next version of Vanilla) and add `is-wrapped` implementation somewhere in custom SCSS.

```scss
.p-code-snippet__block.is-wrapped {
   whitespace: pre-wrap;
}
```

This way only this custom SCSS has to be removed, not inline styles in many places.